custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -942,6 +943,15 @@ def _prepare_docker_command(self, options, run_mode, raw_arguments):
             )
             docker_cmd_args += " -p {port}:{port}".format(port=port)
 
+        if getattr(options, ArgumentOptionsEnvVars.RUNTIME_PARAMS_FILE.lower(), None):
```

Comment:
i think that this doesn't _really_ work either.  I think it would good to have a method

```python
def get_arg_option(options: Namespace, arg_option: ArgumentOptions):
    value_in_namespace = arg_option.lstrip("-").replace("-", "_")
    return getattr(options, value_in_namespace, None)
```

and put this in args_parser and test it and then start using that method instead of hopes and prayers in the Namespace.